frontend/src/pages/auth/signin/styles.css

```diff
@@ -0,0 +1,12 @@
+.sign-in-button {
```

Comment:
Create folder common and move there common styles that can be reused. For example all your buttons should have same style.
Differences will be only for accept or cancel actions